framework/ui/view/qmllaunchprovider.cpp

```diff
@@ -24,6 +24,7 @@ using namespace mu;
 
 static const QString PAGE_TYPE_DOCK("dock");
 static const QString PAGE_TYPE_POPUP("popup");
+static const QString PAGE_TYPE_WIDGET("widget");
```

Comment:
I think, needed rename `popup` now

Reply:
for example?:)

Reply:
same as I suggestion rename type :)
https://github.com/musescore/MuseScore/pull/6361#discussion_r459323269